### PR TITLE
Fixed schedule 

### PR DESCRIPTION
**Pull request by KLIM-nk01**

### CODE REVIEW

File: src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleItem/ScheduleItemStyle.tsx [hunk 1]
```diff
@@ -41,8 +41,8 @@ export const ScheduleItemWrapper = styled.div<IChoiceState>`
   ${({ disabled }) =>
     disabled &&
     `
-    pointer-events: none;
+    // pointer-events: none;
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleItem/ScheduleItemStyle.tsx:44]: remove comments

File: src/components/Profile/Profile/FeatureContainer/DoctorsFeature/OrderManagement/OrderManagement.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ const WorkTimeManagement: React.FC = () => {
       <ContainersName>Patient</ContainersName>
 
       <ContainerContent>
-        {!workTime?.length ? (
+        {!workTime?.filter((appointment) => appointment.patientName).length ? (
```
Comment (katsiaryna-savchuk-itechart) [inline src/components/Profile/Profile/FeatureContainer/DoctorsFeature/OrderManagement/OrderManagement.tsx:14]: maybe better to move into separate const?

File: src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleItem/ScheduleItemStyle.tsx (hunk 1 above)
Comment (KLIM-nk01) [reply src/components/DoctorsPage/DoctorsCard/Schedule/ScheduleItem/ScheduleItemStyle.tsx:44]: fixed

File: src/components/Profile/Profile/FeatureContainer/DoctorsFeature/OrderManagement/OrderManagement.tsx (hunk 2 above)
Comment (KLIM-nk01) [reply src/components/Profile/Profile/FeatureContainer/DoctorsFeature/OrderManagement/OrderManagement.tsx:14]: what do you mean here?

**Review by katsiaryna-savchuk-itechart — APPROVED** [review]

**Pull request merged**